### ISSUE TITLE
Unresolved reference: oneToMany

### ISSUE THREAD

**Issue by Aqluse** [Raizlabs/DBFlow#1404]
Android Studio 3.0 Beta 2
DBFlow Version: 4.0.5
Issue Kind (Bug, Question, Feature): Question

Description: 
Maybe I don't understand something in this life, but I got issue with oneToMany delegated property. It shows me "Unresolved reference" and I don't know why.
![16-08-2017 010222](https://user-images.githubusercontent.com/13728921/29333983-40cd2178-821f-11e7-9d0b-bc2f96f97cc7.png)

Code:
```
@Database(version = DatabaseObject.VERSION, name = DatabaseObject.NAME, backupEnabled = true, consistencyCheckEnabled = true)
object DatabaseObject {
    const val VERSION = 1
    const val NAME = "AppDatabase"
}

@Table(database = DatabaseObject::class)
data class Musician(
        @PrimaryKey(autoincrement = true) var id: Long,
        @Column var artistName: String,
        @Column var itunesID: Long,
        @Column var itunesAMGID: Long,
        @Column var itunesGenre: String
){
    @get:OneToMany(methods = arrayOf(OneToMany.Method.ALL))
    var collections by oneToMany { select from Collection::class where (Collection_Table.musician.artistName.eq(artistName)) }
}

@Table(database = DatabaseObject::class)
data class Collection(
        @PrimaryKey(autoincrement = true) var id: Long,
        @ForeignKey(stubbedRelationship = true) var musician: Musician // ВНИМАНИЕ! Добавление идет не из конкретной коллекции iTunes, а из музыканта, для которого эти коллекции были получены
        )
```

I used an example from page
https://agrosner.gitbooks.io/dbflow/content/Relationships.html
and little edits to support mine tables 

My Gradle:
```
apply plugin: 'com.android.application'
apply plugin: 'kotlin-android'
apply plugin: 'kotlin-android-extensions'
apply plugin: 'kotlin-kapt'

android {
    compileSdkVersion 26
    buildToolsVersion "25.0.3"
    defaultConfig {
        applicationId "com.aqluse.itunesartistfeed"
        minSdkVersion 24
        targetSdkVersion 25
        versionCode 1
        versionName "1.0"
        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
    }
    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
        }
    }
}

dependencies {
    androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
        exclude group: 'com.android.support', module: 'support-annotations'
    })
    compile 'com.android.support:appcompat-v7:26.0.1'
    compile 'com.android.support:design:26.0.1'
    compile 'com.android.support:support-v4:26.0.1'
    compile 'com.android.support.constraint:constraint-layout:1.0.2'
    testCompile 'junit:junit:4.12'
    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"

    // Библиотеки проекта
    compile fileTree(dir: 'libs', include: ['*.jar'])
    compile('com.mikepenz:materialdrawer:5.9.5@aar') {
        transitive = true
    }

    def dbflow_version = "4.0.5"
    annotationProcessor "com.github.Raizlabs.DBFlow:dbflow-processor:${dbflow_version}"
    kapt "com.github.raizlabs.dbflow:dbflow-processor:$dbflow_version"
    compile "com.github.Raizlabs.DBFlow:dbflow-core:${dbflow_version}"
    compile "com.github.Raizlabs.DBFlow:dbflow:${dbflow_version}"
    compile "com.github.Raizlabs.DBFlow:dbflow-kotlinextensions:${dbflow_version}@aar"
}
repositories {
    mavenCentral()
}
```

**Comment by MahdiPishguy** [comment]
did you try my post on stackOverFlow.com?


https://stackoverflow.com/a/45161145/1830228

**Comment by Aqluse** [comment]
@MahdiPishguy how it is suppose to help in my case?

**Comment by agrosner** [comment]
did you import the extension delegate?

**Comment by Aqluse** [comment]
@agrosner after I updated to 4.1.0 from 4.0.5 it become recognizable for import. At the moment I found the issue, 4.0.5 seemed to be a recent version of library and oneToMany wasn't importable at all, but docs contained example with this extension. So problem have been resolved, but I need to check if it really works before closing the issue.

**Comment by agrosner** [comment]
yeah it was released for 4.1
